### PR TITLE
Split generic options in config.mk and introduce `build/bindist`

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: Makefile.linux.mk [hunk 1]
```diff
@@ -87,7 +77,6 @@ clean-submodules:
 	cd frida-python && git clean -xfd
 	cd frida-node && git clean -xfd
 
-
```
Comment (oleavr) [inline Makefile.linux.mk]: The two newlines are intentional and used to visually separate distinct sections.

File: Makefile.mac.mk [hunk 2]
```diff
@@ -47,7 +35,6 @@ HELP_FUN = \
 help:
 	@LC_ALL=C perl -e '$(HELP_FUN)' $(MAKEFILE_LIST)
 
-
```
Comment (oleavr) [inline Makefile.mac.mk]: Let's keep this newline too.

File: Makefile.mac.mk [hunk 3]
```diff
@@ -288,10 +275,18 @@ check-core-mac: build/tmp-mac-i386/frida-core/tests/frida-tests build/tmp-mac-x8
 	build/tmp-mac-i386/frida-core/tests/frida-tests -p $(tests)
 	build/tmp-mac-x86_64/frida-core/tests/frida-tests -p $(tests)
 
-
+BINDIST=build/bindist
```
Comment (oleavr) [inline Makefile.mac.mk]: Left-over?

File: config.mk [hunk 4]
```diff
@@ -0,0 +1,15 @@
+BINDIST=build/bindist
```
Comment (oleavr) [inline config.mk]: Should be spaces around the `=` for consistency with the rest.

**Comment by radare** [comment]
pr updated

**Comment by oleavr** [comment]
Thanks!

**Pull request merged**